types/heic-convert/package.json

```diff
@@ -13,5 +13,15 @@
             "name": "Tregan",
             "githubUsername": "treemmett"
         }
-    ]
+    ],
+    "exports": {
+        ".": {
+            "import": "./index.d.ts",
+            "require": "./index.d.ts"
+        },
+        "./browser": {
+            "import": "./browser.d.ts",
+            "require": "./browser.d.ts"
```

Comment:
this doesn't seem right; browser.d.ts uses `export = convert`, which is TS syntax that applies only to commonjs, meaning that `import: "./browser.d.ts"` shouldn't work.

Also, I don't know whether it's possible to have both import and require point to the same file. I thought that it wasn't.

Reply:
heic-convert itself doesn't have an `exports` in its package.json so DT's package.json should not either

Reply:
You are right, thanks! I was confused but the `heic-convert` package is commonjs so let me update the exports

Reply:
> heic-convert itself doesn't have an exports in its package.json so DT's package.json should not either

Yup, but they have a section with the missing export: https://github.com/catdad-experiments/heic-convert?tab=readme-ov-file#usage-in-the-browser

I guess since their package is commonjs there is no problem using `require('heic-convert/browser')` but when using `import convert from 'heic-convert/browser'` there is an error about missing type definitions, that's why I'm adding the `exports` field